nodejs/express-redis/app/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "express-redis",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "server": "node app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@appsignal/express": "*",
+    "@appsignal/nodejs": "*",
+    "express": "4.17.1",
+    "pug": "*",
+    "redis": "*"
```

Since the app will only work with redis 3, should we lock it to redis 3 so we can actually debug other issues with it?

```suggestion
    "redis": "^3.1.2"
```

When we fix support for version 4, we can lock it to version 4.